policy-utils/src/policy.rs

```diff
@@ -120,6 +122,7 @@ impl Policy {
         execution_strategy: ExecutionStrategy,
         file_hashes: Vec<FileHash>,
         enable_clock: bool,
+        max_memory: u32,
```

Comment:
Maybe instead of `max_memory` make it `max_memory_mb` so users/policy readers/writers aren't left wondering

Reply:
Good idea